### PR TITLE
Documentation for the _api_persist flag

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: core/events.md [hunk 1]
```diff
@@ -109,8 +109,7 @@ Constant           | Event             | Priority |
...
 attribute of an operation](operations.md#recommended-method)):
 
-Listener              | Parameter      | Values         | Default | Description                            |
-----------------------|----------------|----------------|---------|----------------------------------------|
-`ReadListener`        | `_api_receive` | `true`/`false` | `true`  | set to `false` to disable the listener |
-`DeserializeListener` | `_api_receive` | `true`/`false` | `true`  | set to `false` to disable the listener |
-`ValidateListener`    | `_api_receive` | `true`/`false` | `true`  | set to `false` to disable the listener |
+Attribute      | Type   | Default | Description                                                               |
+---------------|--------|---------|---------------------------------------------------------------------------|
+`_api_receive` | `bool` | `true`  | Disables the `ReadListener`, `DeserializeListener` and `ValidateListener` |
```
Comment (soyuka) [inline core/events.md]: > When `false` prevents the following listeners: `ReadListener`, `DeserializeListener` and `ValidateListener`

@teohhanhui expert opinion? :D

Comment (dunglas) [reply core/events.md]: Your call guys.

Comment (teohhanhui) [reply core/events.md]: Set to `false` to disable the ...

Comment (soyuka) [reply core/events.md]: > When `false` disabled the following listeners: ...

**Pull request merged**